pkg/doc/verifiable/credential.go

```diff
@@ -756,12 +756,16 @@ func (vc *Credential) MarshalJSON() ([]byte, error) {
 }
 
 // Presentation encloses credential into presentation.
-func (vc *Credential) Presentation() *Presentation {
+func (vc *Credential) Presentation() (*Presentation, error) {
```

Comment:
Can the name be a bit more descriptive of what the function is actually doing?

Reply:
It's doing an enclosure of VC into VP. Don't know how to name this better...